### DISCUSSION TITLE
Color support

### DISCUSSION THREAD

**Discussion by Timoyoungster** [isontheline/pro.webssh.net#1075] [Q&A]
Hello,
I have a question regarding WebSSH's handling of colors.
I redid my neovim config today and with that switched my colortheme from rose-pine to tokyonight-night.
Unfortunately I now have the problem that when using nvim I do not see the cursor (block mode). Additionally I realized that when using tmux the tmux statusline is not the typical vibrant green, but more a sort of washed out, almost gray green. The positive side is that when using nvim from inside there the cursor is visible again, but the colors are not how they're supposed to be.

`echo $TERM` from the shell returns 'xterm-256color'
`:echo &term` from vim (started with shell) returns 'xterm-256color'
`:echo &term` from vim (started with tmux) returns 'tmux-256color'

I did all those config changes on my macbook (iTerm to ssh into the server which I am using as my dev machine) and had no color issues there, so I assume it has something to do with WebSSH's color handling. (Does it maybe not support 256 colors)

Does anyone know how I can get all the colors to be displayed correctly in every case?

**Comment by isontheline** [comment]
Hello @Timoyoungster 👋 

WebSSH should support 256 colors out of the box.

I think the trouble is related how I manage custom theme colors.

Does the colors you see are the same as ones displayed on the WebSSH Color Theme chosen on WebSSH settings?

**Reply by Timoyoungster** [reply]
I chose tokyonight for both, so they should be the same. Could it be something with truecolor support?

**Reply by isontheline** [reply]
Perhaps.

To be sure, could you share some configuration?
I would like to reproduce as much as possible.

**Reply by Timoyoungster** [reply]
I use Lazy.nvim.
This is the part where I install and set up the theme.

`{
    'folke/tokyonight.nvim',
    lazy = false,
    priority = 1000,
    config = function()
        vim.cmd.colorscheme 'tokyonight-night'
        vim.cmd.hi 'Comment gui=none'
    end,
},`

other than that, I have no configuration regarding color things. Everything on defaults (tmux doesn't have a config at all yet).

If you want to have a closer look I have my whole neovim config in a public repo: https://github.com/Timoyoungster/nvim-config-V2.

**Reply by isontheline** [reply]
Thanks @Timoyoungster 
I need some time to reproduce
